website/src/components/playground-component/import-map.ts

```diff
@@ -26,11 +35,10 @@ export async function loadImportMap({ latestVersion }: { latestVersion: string }
 
   (window as any).importShim.addImportMap(importMap);
 
-  await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
this feels like it was added for testing and never removed but, will double check with the published version there is no timing issues

Reply:
seems to work fine